TrackletAlgorithm/MatchEngineUnit.h

```diff
@@ -10,7 +10,11 @@
 #include "AllStubMemory.h"
 #include "AllProjectionMemory.h"
 #include "FullMatchMemory.h"
-#include "hls_math.h"
```

Comment:
@carriganm95 I just checked, and we don't actually need to include hls_math.h in any of the modules, so you can just remove this include statement everywhere you find it.

The only exception is this file, where you need to replace it with `#include <cassert>`.